### ISSUE TITLE
Accessibility function stopped working after One UI 7.0 update

### ISSUE THREAD

**Issue by Ravn7** [Genymobile/scrcpy#6319]
## Environment
 - Windows 11 24H2
 - scrcpy 3.3.1
 - Windows release
 - Galaxy A55 
 - One UI 7.0
 - Android 15
 
Since One UI 7.0 update, "Pinch zoom" accessibility function doesn't work. You can find it here: Settings - Accessibility - Interaction and dexterity - Assistant menu - Select Assistant menu items. 

It doesn't work at all. When you click it, you simply can't use it. Although it works with AirDroid, which means I have to launch it every single time I want to zoom in or out of a picture. And it doesn't work with Vysor and Phone Link.

**Comment by huseyin-ace** [comment]
You're Right.My Needs Are:

Windows 11 24h2

scrpy 3.3.1

Galaxy A54

One UI 7.0

Android 15
